package.json

```diff
@@ -85,6 +85,7 @@
     "start:ci": "env-cmd --no-override -e ci,b2c,$SPA_ENV nx serve storefrontapp --configuration=development",
     "start:ci:b2b": "env-cmd --no-override -e ci,b2b,$SPA_ENV nx serve storefrontapp --configuration=development",
     "start:opf": "env-cmd --no-override -e opf,$SPA_ENV nx serve storefrontapp --configuration=development --ssl",
+    "start:opf:vm": "env-cmd --no-override -e opf,$SPA_ENV nx serve storefrontapp --configuration=development",
```

Comment:
need to run opf without ssl on azure vm to test ApplePay,. VM already has ssl with nginx.